docs/how-to-guides/Interact-with-gnoland.md

```diff
@@ -0,0 +1,94 @@
+---
+id: interact-with-gnoland
+---
+
+# Interact with Gno.land
+
+This tutorial will teach you how to interact with the Gnoland blockchain by creating an account and calling various realms to send transactions on the network.
+
+## Prerequisites
+
+- [Installation](../getting-started/installation.md)
+
+## Create an Account
+
+In order to interact with Gnoland, you need an account that you will use to sign and send transactions. You may create a new account with `gnokey generate` or recover an existing one with `gnokey add`. Confirm that your account was successfully added with `gnokey list` to display all accounts registered in the key base of your device.
+
+```bash
+gnokey generate # create a new seed phrase (mnemonic)
+
+gnokey add -recover {your_account_name} # registers a key with the name set as the value you put in {your_account_name} with a seed phrase
+
+gnokey list # check the list of keys
+```
+
+## Register As a User
+
+```bash
+gnokey maketx call \
+    -gas-fee="1ugnot" \
+    -gas-wanted="5000000" \
+    -broadcast="true" \
+    -remote="test3.gno.land:36657" \
```

Comment:
I recommend using staging.gno.land instead of test3 for all the documentation. Unlike test3, staging.gno.land is consistently updated with git. It's worth noting that staging always has a faucet, whereas test3 can run out of resources. Ideally, test3 should serve as a more experimental testnet, while staging provides a safe and user-friendly environment to discover gno.